input/intro-notes/StructureDefinition-au-core-immunization-intro.md

```diff
@@ -15,22 +15,24 @@ A resource conforming to this profile is conformant to:
 
 Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://build.fhir.org/ig/HL7/fhir-ipa), [International Patient Summary](http://build.fhir.org/ig/HL7/fhir-ips), or [US Core](http://hl7.org/fhir/us/core) **MAY NOT** conform to AU Core.
 
-
-### Profile specific implementation guidance
```

Comment:
Restore the deleted heading and implementation guidance

Reply:
Done